--- librawnet-devel-1.0.1/src/rawnetcc.c	2010-09-23 15:48:15.000000000 +0200
+++ librawnet-devel-1.0.1/src/rawnetcc.c.new	2010-09-23 15:57:40.000000000 +0200
@@ -152,7 +152,7 @@
      It must be executed with execv() because of suid bit */
   int setcap_argc = 3;
   char* setcap_argv[setcap_argc + 1]; /* +1 for NULL */
-  setcap_argv[0] = "/sbin/setcap";
+  setcap_argv[0] = "/usr/sbin/setcap";
   setcap_argv[1] = "cap_net_raw,cap_net_admin=eip";
   setcap_argv[2] = exec_file;
   setcap_argv[3] = NULL;
